src/main/java/org/jabref/gui/preferences/general/GeneralTab.java

```diff
@@ -141,8 +146,28 @@ void importTheme() {
     @FXML
     private void openBrowser() {
         String url = "https://themes.jabref.org";
+
+        FilePreferences filePreferences = null;
+
+        String userAndHost = filePreferences.getUserAndHost();
+        boolean storeFilesRelativeToBibFile = filePreferences.shouldStoreFilesRelativeToBibFile();
+        String fileNamePattern = filePreferences.getFileNamePattern();
+        String fileDirectoryPattern = filePreferences.getFileDirectoryPattern();
+        boolean downloadLinkedFiles = filePreferences.shouldDownloadLinkedFiles();
+        boolean fulltextIndexLinkedFiles = filePreferences.shouldFulltextIndexLinkedFiles();
+        Path workingDirectory = filePreferences.getWorkingDirectory();
+        ObservableSet<ExternalFileType> externalFileTypes = filePreferences.getExternalFileTypes();
+        boolean createBackup = filePreferences.shouldCreateBackup();
+        Path backupDirectory = filePreferences.getBackupDirectory();
+
+        FilePreferences newFilePreferences = new FilePreferences(
```

Comment:
You don't need to construct new file preferences 
you can simply use  preferencesService.getFilePreferences()

That should fix your errors

Reply:
It worked! The errors are gone, and everything is running smoothly now. Thanks a bunch for your help!